backend/src/chats/chat-user-permissions/chat-user-permission.spec.ts

```diff
@@ -145,60 +134,65 @@ describe('Chat - User - Permission relationship', () => {
 			{
 				chat_id: chat_ids[0],
 				user_id: user_ids[0],
-				permission_id: permission_ids[0],
+				permission: permissionsEnum.READ,
 			},
 			{
 				chat_id: chat_ids[0],
 				user_id: user_ids[1],
-				permission_id: permission_ids[0],
+				permission: permissionsEnum.READ,
```

Comment:
what is the difference between the two toHaveLengtth above?

Reply:
this one was difficult to figure out... (what you meant by the question)
but I guess:

the test: `it('should allow one user to be in multiple chats'`
checks what it says it does
and the test: `it('should allow one chat to have multiple users'`
also

so one is 1 chat 2 users
the other is 2 chats same user

Reply:
sorry the to have length refers apperantly to another line: the function tohavelength() 
and besides I was brainfarting 
sorry I dont know why I didnt understand this, 
lets pretend this didnt happen